csvcoder/csvcoder_file.go

```diff
@@ -36,14 +35,20 @@ type FileParser struct {
 	fatalErr error
 }
 
+// recordsReader is an interface of a reader that reads raw records from the
+// data source. For example, csv.Reader is a recordsReader.
+type recordsReader interface {
```

Comment:
The interface should be exported so its type may be used by clients. I would also prefer to rename it "RowReader."